bin/setup

```diff
@@ -8,14 +8,17 @@ if [ "$CI" ]; then
 fi
 
 # Make sure we have Bundler installed
-gem install bundler -v 1.17.3 --conservative
+gem install bundler --conservative
 
 # Set up Ruby dependencies via Bundler into .bundle folder
 rm -f .bundle/config
 
 bundle check --path .bundle > /dev/null 2>&1 ||
   bundle install --path=.bundle $BUNDLER_ARGS
 
+# Set up node.js modules
+yarn install
```

Comment:
Could we use `bin/yarn`?

Reply:
I think it's not correct, yarn is not ruby gem, it is system-wide installed package manger, so there is no reason to have script at bin/ to just run it.